diff --git a/src/component/shared.ts b/src/component/shared.ts
index 371db8bbf5e060bbfbfcdd422a1d054e20ee1537..3bcc94492bfc1c8b6b0e7c97e6d2d000a528a94b 100644
--- a/src/component/shared.ts
+++ b/src/component/shared.ts
@@ -70,7 +70,7 @@ export const DEFAULT_RETRY_BEHAVIOR: RetryBehavior = {
   base: 2,
 };
 // This ensures that the type satisfies the schema.
-const _ = {} as RetryBehavior satisfies Infer<typeof retryBehavior>;
+void ({} as RetryBehavior satisfies Infer<typeof retryBehavior>);
 
 export const vResult = vRunResult(v.any());
 export function vRunResult<RV extends Validator<any, any, any> = VAny>(
